helm/adelphi/templates/nosqlbench-jobs.yaml

```diff
@@ -38,11 +41,15 @@ spec:
     - name: workload
       configMap:
         name: nosqlbench-workload-configmap
+    - name: results-pv
+      persistentVolumeClaim:
+        claimName: results-pvc
   - name: nosqlbench-target
     container:
       image: nosqlbench/nosqlbench:latest
       args: [
         "--show-stacktraces",
+        "--report-csv-to=/results/nosqlbench-target",
```

Comment:
Same question re: is this just a CSV file (see above for more detail)

Reply:
No, it's a folder.